frontend/src/components/Modal/RiderModalInfo.tsx

```diff
@@ -124,10 +112,29 @@ const RiderModalInfo = ({
             <p className={styles.error}>Please enter a phone number</p>
           )}
         </div>
-        <div className={cn(styles.gridR2, styles.gridCBig1)}>
+        <div className={cn(styles.gridR2, styles.gridCSmall1)}>
+          <Label className={styles.label} htmlFor="address">
+            Address:{' '}
+          </Label>
+          <Input
+            id="address"
+            name="address"
+            type="text"
+            ref={register({
+              required: true,
+              pattern: /^[a-zA-Z0-9\s,.'-]{3,}$/,
+            })}
+          />
+          {errors.address && (
+            <p className={styles.error}>Please enter an address</p>
+          )}
+        </div>
+
+        <div className={cn(styles.gridR2, styles.gridCSmall2)}>
           <Label className={styles.label} htmlFor="needs">
             Needs:{' '}
           </Label>
+          <br />
           <select name="needs" ref={register({ required: true })}>
```

Comment:
maybe you could add some styling here (border-radius, increase width and height) to have the select input's size match all the other inputs' sizes